test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -7,9 +7,10 @@ Feature: Bind a single application to multiple services
     Background:
         Given Namespace [TEST_NAMESPACE] is used
         * Service Binding Operator is running
+        * OLM Operator "backend" is running
```

Comment:
is this fake operator really required for the scenarios in this feature file to pass? My personal feeling is no - what most of the scenarios need is just existence of appropriate CRD so that service instances can be created. We should really identify scenarios where operators ClusterServiceVersion is needed and only then use this step. For all other, we should define another step, e.g. `Operator X is running` where we just deploy needed CRDs.

Reply:
I agree with @pedjak, we should depend on CSV only if that is essential for the scenario itself

Reply:
Okay, I removed `Operator X is running` step from various scenarios PTAL.